aggregator/pom.xml

```diff
@@ -159,7 +159,9 @@
                                 <mkdir dir="${oldClassesDir}"/>
 
                                 <exec executable="diff"
-                                      resultproperty="diff.result">
+                                      resultproperty="diff.result"
+                                      discardOutput="${nonfail.errors.quiet}"
+                                      discardError="${nonfail.errors.quiet}">
```

Comment:
Rather than suppress errors by default, does it make more sense to make this execution conditional on old class dir existing or some other check that indicates this is not a clean build?